src/components/Board/Board.container.js

```diff
@@ -193,7 +194,6 @@ export class BoardContainer extends Component {
     isSelecting: false,
     isLocked: true,
     tileEditorOpen: false,
-    translatedBoard: null,
```

Comment:
I love seeing a bunch of code getting deleted 😍

Reply:
Hehe was a pleasure to press 'delete'